exceptions.txt

```diff
@@ -419,3 +419,6 @@
 294973: 'Home Fires UK',
 295438: 'Shark UK',
 148581: 'Strike Back Legacy',
+281714: 'Ballers', 'Ballers (2015)', 'Ballers (2014)',
```

Comment:
I removed Ballers so it doesn't search 2x ... Now, how would this list go about being merged with the "main one".